src/dappsExplorer/NoResults.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import { Trans, useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import { DappExplorerEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import Touchable from 'src/components/Touchable'
+import { DappFilter } from 'src/dapps/types'
+import Colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+
+interface Props {
+  filter: DappFilter | null
+  removeFilter: () => void
+  testID: string
+}
+
+export function NoResults({ filter, removeFilter, testID }: Props) {
+  const { t } = useTranslation()
+
+  const removePress = () => {
```

Comment:
can we stick to the vanilla naming conventions so that it's really easy to guess what's happening 🙈 i think generally we use `onPress` or `onPressXXXX` or `handlePressXXXX` for click handers, and the rest for executing logic

Reply:
Fixed in e5341bcc2662f046e048379299861b43e3205765!